sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpClient.cs

```diff
@@ -92,7 +93,8 @@ internal class AmqpClient : TransportClient
         internal AmqpClient(
             string host,
             ServiceBusTokenCredential credential,
-            ServiceBusClientOptions options)
+            ServiceBusClientOptions options,
+            bool useTls = true)
```

Comment:
Nit: any reason to not make it required?

Reply:
Fair question.   The intention was to avoid test churn, but on second glance, there are only a few referencing.   Updated.